libs/blocks/modal/modal.js

```diff
@@ -199,7 +199,12 @@ export default function init(el) {
 window.addEventListener('hashchange', (e) => {
   if (!window.location.hash) {
     const url = new URL(e.oldURL);
-    const dialog = document.querySelector(`.dialog-modal${url.hash}`);
+    let dialog;
+    try {
+      dialog = document.querySelector(`.dialog-modal${url.hash}`);
+    } catch (error) {
+      /* do nothing */
+    }
     if (dialog) closeModal(dialog);
```

Comment:
I guess this can be moved inside the `try` block and have `dialog` as a `const` directly